resources_pages/quiz.md

```diff
@@ -1,12 +1,15 @@
 ---
 layout: page
-title: MDS quizzes
+title: MDS Quizzes
 ---
 
+## Everything MDS Quiz Related
 
 ### Overview
 
-With the exception of [project courses](https://ubc-mds.github.io/2019-08-22-project-courses/), MDS courses will have two quizzes throughout the block, typically one in the middle of the course and another after the course ends during the free days between blocks. The quizzes for a course will appear on the [MDS calendar](https://ubc-mds.github.io/calendar/) (ideally before the course starts) so that students are aware of the timing. We anticipate most quizzes being 50 minutes long.
+With the exception of [project courses](https://ubc-mds.github.io/2019-08-22-project-courses/), MDS courses will have two quizzes throughout the block, typically one in the middle of the course and another after the course ends during the free days between blocks. Each quiz will last for 50 minutes and will be taken inside of the CBTF at UBC. You will also have the option to attend a review quiz session which allows you to look back at your quiz to see where you might have made mistakes. Please be sure to read through the [quiz policy](https://ubc-mds.github.io/policies/#quiz-policies) as well. The information in the quiz policy page has the final say in any discrepancies between the two pages.
```

Comment:
I think this website no longer contains an exhaustive list of project courses. DSCI 541 and DSCI 542 also don't have quizzes.

Reply:
We can remove the link. I am not aware of any updated link.